array_api_compat/common/_helpers.py

```diff
@@ -99,7 +99,7 @@ def your_function(x, y):
                 namespaces.add(torch)
         else:
             # TODO: Support Python scalars?
-            raise TypeError("The input is not a supported array type")
+            raise TypeError(f"{type(x)!r} is not a supported array type")
```

Comment:
`repr(type(x))` is like `<class 'int'>`. You want something like `type(x).__name__`.

Reply:
```suggestion
            raise TypeError(f"{type(x).__name__} is not a supported array type")
```